.github/actions/create-default-e2e-config-override/action.yml

```diff
@@ -0,0 +1,149 @@
+name: 'Create Default E2E Test Config Override From Inputs'
+inputs:
+  chainlink_image:
+    required: false
+    description: 'Chainlink image to use for the tests. Example: "chainlink/chainlink:develop"'
+  chainlink_version:
+    required: false
+    description: 'Chainlink version to use for the tests. Example: "v2.10.0" or sha'
+  chainlink_postgres_version:
+    required: false
+    description: 'Chainlink Postgres version to use for the tests'
+  chainlink_upgrade_image:
+    required: false
+    description: 'Chainlink upgrade image to use for the tests'
+  chainlink_upgrade_version:
+    required: false
+    description: 'Chainlink upgrade version to use for the tests'
+  selected_networks:
+    required: false
+    description: 'Selected networks for the tests (comma separated)'
+  pyroscope_server:
+    required: false
+    description: 'Pyroscope server for the tests'
+  pyroscope_environment:
+    required: false
+    description: 'Pyroscope environment for the tests'
+  pyroscope_key:
+    required: false
+    description: 'Pyroscope key for the tests'
+  test_log_collect:
+    required: false
+    description: 'Whether to always collect test logs'
+  loki_run_id:
+    required: false
+    description: 'Loki run ID for the tests'
+  loki_endpoint:
+    required: false
+    description: 'Loki endpoint for the tests'
+  loki_tenant_id:
+    required: false
+    description: 'Loki tenant ID for the tests'
+  loki_basic_auth:
+    required: false
+    description: 'Loki basic authentication for the tests'
+  logstream_log_targets:
+    required: false
+    description: 'Logstream log targets for the tests (comma separated)'
+  grafana_url:
+    required: false
+    description: 'Grafana URL for the tests'
+  grafana_dashboard_url:
+    required: false
+    description: 'Grafana dashboard URL for the tests'
+  grafana_bearer_token:
+    required: false
+    description: 'Grafana bearer token for the tests'
+  private_ethereum_network_execution_layer:
+    required: false
+    description: 'Private Ethereum network execution layer for the tests'    
+  private_ethereum_network_custom_docker_image:
+    required: false
+    description: 'Private Ethereum network custom Docker image for the tests'
```

Comment:
these two worry me in the context of CCIP, which might be passing here multiple (comma-separated?) values? have you checked whether this structure fits into what CCIP needs?